src/test/java/ikysil/training/dao/InMemoryOrderDaoTest.java

```diff
@@ -19,35 +19,35 @@
     private OrderDto order2 = new OrderDto("3257", "twenty two", "none", "4024", "seconduser@gmail.com");
 
     @Before
-    public void saveOrderTest(){
+    public void insertOrders(){
         dao.saveOrder(order1);
         dao.saveOrder(order2);
     }
 
     @Test
     public void findOrderTest(){
         OrderDto testOrder1 = dao.findOrder("1144");
-        assertEquals(order1, testOrder1);
+        assertThat(order1).isEqualTo(testOrder1);
 
         OrderDto testOrder2 = dao.findOrder("3257");
-        assertEquals(order2, testOrder2);
+        assertThat(order2).isEqualTo(testOrder2);
     }
 
     @Test
     public void updateOrderTest() {
         OrderDto newOrder = new OrderDto("3257","thirty three", "noneone", "2440", "newuser@gmail.com");
         dao.updateOrder("3257", newOrder);
-        assertEquals(dao.findOrder("3257"), newOrder);
+        assertThat(dao.findOrder("3257")).isEqualTo(newOrder);
     }
 
     @Test
     public void findAllTest(){
         List<OrderDto> orders = dao.findAll();
-        assertEquals(2, orders.size());
-        deleteOrderTest();
+        assertThat(orders.size()).isEqualTo(2);
```

Comment:
assertJ has this kind of assertion for lists:
```
assertThat(orders).hasSize(2)
```
agree it is more human readable